kolibri/utils/lru_cache.py

```diff
@@ -0,0 +1,185 @@
+try:  # NOQA
```

Comment:
This file reappeared, was gone before... we can perhaps optimize some more functions using it.